src/scaffolding/wizard/netCore/NetCoreGatherInformationStep.ts

```diff
@@ -70,11 +71,26 @@ export class NetCoreGatherInformationStep extends GatherInformationStep<NetCoreS
             wizardContext.serviceName = getValidImageNameFromPath(wizardContext.artifact);
         }
 
+        if (!wizardContext.dockerBuildContext) {

```

Comment:
This is redundant since the `super` call would set this. Can probably get rid of this and leave a comment.